android/java/org/chromium/chrome/browser/app/domain/PortfolioModel.java

```diff
@@ -79,9 +80,9 @@ public PortfolioModel(Context context, TxService txService, KeyringService keyri
         mBraveWalletService = braveWalletService;
         mAssetRatioService = assetRatioService;
         mSharedData = sharedData;
-        _mNftModels = new MutableLiveData<>(Collections.emptyList());
+        _mNftModels = new MutableLiveData<>();
         mNftModels = _mNftModels;
-        _mIsDiscoveringUserAssets = new MutableLiveData<>(false);
+        _mIsDiscoveringUserAssets = new MutableLiveData<>();
```

Comment:
Without the initial value we call the observers less times, and loading is much faster 🚀

Reply:
We should prefer to provide an initial value if possible to avoid null values and check. Don't think there's any API restriction to cause any delay here.